lib/types/falsy.type.ts

```diff
@@ -0,0 +1 @@
+type Falsy = '' | 0 | false | null | undefined;
```

Comment:
This seems to be unused, and fails linting with
~~~~
ERROR: ..../node-oauth2-server/lib/types/falsy.type.ts:1:14 - Union type cannot include both 'null' and 'undefined'.
~~~~

https://palantir.github.io/tslint/rules/no-null-undefined-union/ seems to be the default, and in any case this file can just be removed without ill effects it seems.